ballroom/src/main/java/org/jboss/hal/ballroom/form/SwitchItem.java

```diff
@@ -25,6 +25,7 @@
 import org.jboss.hal.resources.CSS;
 import org.jboss.hal.resources.Ids;
 
+import static org.jboss.gwt.elemento.core.Elements.i;
 import static org.jboss.gwt.elemento.core.Elements.*;
```

Comment:
Looks like there is a check-style failure here:
ImportOrder: Wrong order for 'org.jboss.gwt.elemento.core.Elements.*' import. [1]
[1] https://ci.wildfly.org/viewLog.html?buildId=147228&buildTypeId=hal_Console&tab=buildLog

Reply:
Thanks @MMarus. Fixed by https://github.com/hal/console/commit/e1b68b04da8eda8ed28c2a674bea3017d65478a5